src/main/java/com/uid2/shared/store/KeysetKeyStoreSnapshot.java

```diff
@@ -30,7 +30,7 @@ public List<KeysetKey> getActiveKeysetKeys() {
     public KeysetKey getActiveKey(int keysetId, Instant now) {
         List<KeysetKey> keysetKeys = keysetMap.get(keysetId);
         if(keysetKeys == null || keysetKeys.isEmpty()) return null;
-        int keysetKeysIndex = Utils.upperBound(keysetKeys, now, (ts, k) -> ts.isBefore(k.getActivates()));
+        int keysetKeysIndex = Utils.upperBound(keysetKeys, now, (ts, k) -> !ts.isAfter(k.getActivates())); // before || equal
```

Comment:
Why did you make this change? Can you explain the difference in logic here? Looks to be the same logic just more complex

Reply:
also if this change is intentional, let's add a test case